launch/dynamo-run/src/input/http.rs

```diff
@@ -98,6 +98,14 @@ pub async fn run(
             manager.add_completions_model(model.service_name(), cmpl_pipeline)?;
         }
     }
+    tracing::debug!(
+        "Supported routes: {:?}",
+        http_service
+            .route_docs()
+            .iter()
+            .map(|rd| rd.to_string())
+            .collect::<Vec<String>>()
+    );
```

Comment:
nit: Can we put these on different lines? It will make it easier to read when there are many of them as this project grows.

Reply:
i.e.
```
"GET /metrics"
"GET /dynamo/alpha/list-models"
"GET /v1/models"
"POST /v1/chat/completions"
"POST /v1/completions"
etc.

Reply:
Having each log be a single line make it much easier to deal with once we aggregate the logs in something like [Loki](https://grafana.com/oss/loki/). It's a tradeoff between humans and machines :-/

Reply:
I didn't want to be too prescriptive here. As long as each endpoint is written on its own line, whether that's a new log line or formatted in that way, that would be fine with me. 
```
// possible pseudo code
for (endpoint in endpoints) {
  LOGGER.info("Loaded endpoint: {endpoint}")
}
```

Reply:
There are only five endpoints right now which is fine as a list, so we can cross this bridge when we get there. It is
 important that each log line is complete and cohesive, but we can discuss splitting later.